### ISSUE TITLE
Error on installing using Yarn

### ISSUE THREAD

**Issue by BrOrlandi** [zouhir/eslint-config-standard-preact#13]
I'm not being able to install it via Yarn.

**How to reproduce:**

`yarn add eslint-config-standard-preact`

**Result:**

```sh
[3/4] 🔗  Linking dependencies...
error An unexpected error occurred: "could not find a copy of eslint to link in /project/
node_modules/eslint-config-standard-preact/node_modules".
```

**System info:**
MacOS Mojave version 10.14.1 (18B75)
Node Version: tried on v8.12.0 and v9.11.2
Yarn version: 1.12.1

**Comment by tonydtran** [comment]
Same here. Any solution? I'm afraid this project is not maintained anymore :/

This fork seems to be updated: https://github.com/bluesea365/eslint-config-standard-preact

**Comment by goferito** [comment]
I get the same error, but I'm not using this module. My guess is that it's related to eslint or babel-esling

**Comment by lkmill** [comment]
@BrOrlandi upgrading to yarn 1.15.3 solved this for me.

@zouhir you should probably update the dependencies of this package
